lib/code_corps/github/event/issues/changeset_builder.ex

```diff
@@ -51,18 +54,20 @@ defmodule CodeCorps.GitHub.Event.Issues.ChangesetBuilder do
     |> MarkdownRendererService.render_markdown_to_html(:markdown, :body)
     |> Changeset.put_change(:created_from, "github")
     |> Changeset.put_change(:modified_from, "github")
+    |> Changeset.put_change(:github_issue_id, github_issue_id)
     |> Changeset.put_change(:github_repo_id, github_repo_id)
     |> Changeset.put_change(:project_id, project_id)
     |> Changeset.put_change(:task_list_id, task_list_id)
     |> Changeset.put_change(:user_id, user_id)
     |> Changeset.validate_required([:project_id, :task_list_id, :title, :user_id])
+    |> Changeset.assoc_constraint(:github_issue)
     |> Changeset.assoc_constraint(:github_repo)
     |> Changeset.assoc_constraint(:project)
     |> Changeset.assoc_constraint(:task_list)
     |> Changeset.assoc_constraint(:user)
```

Comment:
Considering the amount of associations here, maybe we should go with 

```Elixir
|> Changeset.put_change(:github_issue_id, github_issue_id)
|> Changeset.assoc_constraint(:github_issue)
```

etc.

Might be easier to keep track of these that way and ensure they all have the proper assoc constraint. I'm not strongly for one or the other approach, but wanted to mention the idea.